Formula/bupstash.rb

```diff
@@ -0,0 +1,36 @@
+class Bupstash < Formula
+  desc "Easy and efficient encrypted backups"
+  homepage "https://bupstash.io"
+  url "https://github.com/andrewchambers/bupstash/releases/download/v0.10.2/bupstash-v0.10.2-src+deps.tar.gz"
```

Comment:
What deps does this bundle?

Reply:
@carlocab, I just force pushed with your suggestion. Thank you!

> What deps does this tarball bundle?

The tarball bundles 96 cargo crates, most of which have not been packaged by homebrew:

```
ahash				indicatif			rusqlite
aho-corasick			instant				ryu
anyhow				itoa				same-file
arrayref			lazy_static			scopeguard
arrayvec			libc				serde
atty				libsqlite3-sys			serde_bare
autocfg				lock_api			serde_derive
bitflags			lz4				serde_json
blake3				lz4-sys				serial_test
cc				memchr				serial_test_derive
cfg-if				nix				shlex
cfg-if-0.1.10			num-integer			smallvec
chrono				num-traits			syn
codemap				number_prefix			tar
codemap-diagnostic		once_cell			tempfile
console				openat				termcolor
constant_time_eq		parking_lot			terminal_size
crossbeam-channel		parking_lot_core		thiserror
crossbeam-utils			path-clean			thiserror-impl
digest				pkg-config			time
encode_unicode			ppv-lite86			typenum
fallible-iterator		proc-macro2			unicode-width
fallible-streaming-iterator	quote				unicode-xid
filetime			rand				vcpkg
generic-array			rand_chacha			version_check
getopts				rand_core			walkdir
getrandom			rand_hc				wasi
glob				rangemap			winapi
hashbrown			redox_syscall			winapi-i686-pc-windows-gnu
hashlink			regex				winapi-util
hermit-abi			regex-syntax			winapi-x86_64-pc-windows-gnu
humantime			remove_dir_all			xattr
```